scylla-server-rust/tests/data_service_test.rs

```diff
@@ -0,0 +1,128 @@
+#[path = "test_utils.rs"]
+mod test_utils;
+
+use prisma_client_rust::QueryError;
+use protobuf::SpecialFields;
+use scylla_server_rust::{
+    controller::data_controller::DataSend,
+    serverdata::ServerData,
+    services::{data_service, data_type_service, node_service, run_service},
+};
+use test_utils::cleanup_and_prepare;
+
+const DATA_TYPE_NAME: &str = "test";
+
+#[tokio::test]
+async fn test_data_service() -> Result<(), QueryError> {
+    let db = cleanup_and_prepare().await?;
+
+    run_service::create_run_with_id(&db, 0, 0).await?;
+    node_service::upsert_node(&db, DATA_TYPE_NAME.to_owned()).await?;
```

Comment:
creating a node with a data type name?

Reply:
oh thats just me being lazy not wanted to use the string "test".  Ill fix that.